examples/readme.md

```diff
@@ -0,0 +1,39 @@
+# Yew Examples
+
+Use `build.sh` script to build examples. 
+
+Examples are prepared for `wasm-bindgen` except `minimal_wp` which is prepared for `wasm-pack`.
```

Comment:
Can you explain that `wasm-bindgen` was chosen because:
1. Want to keep setup simple (no bundlers)
2. Prefer to keep examples as binaries rather than libraries (wasm-pack limitation)

Also, I think here would be a good place to link to https://yew.rs/docs/getting-started/starter-templates for people who are interested in how to setup their own app after they learn about examples.

Reply:
This part is little bit confusing for me. 

In https://yew.rs/docs/getting-started/project-setup is written: `We don't recommend using wasm-bindgen directly because it requires hand-writing some JavaScript to bootstrap you WebAssembly binary.`

But wasm-pack and wasm-bindgen both can use same `/examples/static/index.html` file. No other steps are needed. Neither of them needs to use bundler when compiled with `--target web`

Reply:
🤔 you're right. The docs need to be updated. I wonder if I wrote those before `--target web` or something.. or maybe I just wrote incorrectly 😅